.github/renovate.json

```diff
@@ -1,6 +1,41 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended"
+    "config:base"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "lockFileMaintenance": {
+    "enabled": true
+  },
+  "prCreation": "not-pending",
+  "rangeStrategy": "update-lockfile",
+  "stabilityDays": 3,
+  "github-actions": {
+    "fileMatch": [
+      "^ci\\/.*/[^/]+\\.ya?ml$"
+    ]
+  },
+  "packageRules": [
+    {
+      "matchManagers": [
+        "cargo"
+      ],
+      "matchUpdateTypes": [
+        "patch"
+      ],
+      "enabled": false
```

Comment:
I'm a little confused by both of these `packageRules` being disabled. What exactly does this mean?

Reply:
I think this was done upstream in https://github.com/rust-lang/rustup/pull/3632 and https://github.com/rust-lang/rustup/pull/3640

My reading is that it's disabling the automatic cargo update PRs for `patch` updates and `minor` updates that don't match the `matchCurrentVersion` expression. I think we're happy for both of those to create automated PRs so maybe we should go back to defaults here? 

I was operating under the assumption the way things are working in Rustup is to your liking and so we should match configs but maybe that was the wrong way to think about this.

Reply:
Ah yes, I think this disabling updates for stuff that is covered by the `lockFileMaintenance.enabled = true` stuff.